working-offline.md

```diff
@@ -0,0 +1,37 @@
+# Working Offline
+<!--Explain a bigger picture of sync on/off feature and how to switch-->
+The OpenMRS Android Client can also be successfully used in areas with limited connectivity. As you open the OpenMRS Android Client the login screen appears. Here you may provide your login credentials and a location (e.g. "Registration Desk")
+To use the client in the offline mode you may press the option just beside the word "online" to switch to the offline mode. 
+
+*Using Offline Mode requires a previous sign-in with the credentials provided, and thus is disabled on initial login attempts*
+    
+![OpenMRS Android Client Offline login screen](assets/openmrs_android_client_offline_login.png)
+
+
+1. Registering Patients
```

Comment:
Can you make this into a header? It will be easier for users to read. Also, remove the numbering, it won't be needed.